.i18n/Game-zh_CN.po

```diff
@@ -0,0 +1,5296 @@
+
+#: Game.Levels.Implication.L05succ_inj2
+msgid ""
+"Applying a proof of $P\\implies Q$ to the *goal* changes $Q$ to $P$.\n"
+"Now try `rw [succ_eq_add_one]` to make the goal more like the hypothesis."
+msgstr ""
+"应用一个 $P\\implies Q$ 的证明到*目标*上，会将 $Q$ 变为 $P$。\n"
+"现在尝试使用 `rw [succ_eq_add_one]` 来使目标更像假设。"
+
+#: Game.Levels.Implication.L05succ_inj2
+msgid ""
+"Now rewrite `four_eq_succ_three` backwards to make the goal\n"
+"equal to the hypothesis."
+msgstr "现在反方向重写 `four_eq_succ_three` 使目标等于假设。"
+
+#: Game.Levels.Implication.L05succ_inj2
+msgid "You can now finish with `exact h`."
+msgstr "现在您可以使用 `exact h` 来完成证明。"
+
+#: Game.Levels.Implication.L05succ_inj2
+msgid ""
+"Many people find `apply t at h` easy, but some find `apply t` confusing.\n"
+"If you find it confusing, then just argue forwards.\n"
+"\n"
+"You can read more about the `apply` tactic in its documentation, which you can view by\n"
+"clicking on the tactic in the list on the right."
+msgstr ""
+"许多人觉得 `apply t at h` 很容易，但有些人觉得 `apply t` 令人困惑。\n"
+"如果你觉得很困惑，那就用前一种吧。\n"
+"\n"
+"您可以在其文档中阅读有关 `apply` 策略的更多信息，您可以通过\n"
+"单击右侧列表中的策略的方式查看。"
+
+#: Game.Levels.Implication.L06intro
+msgid "intro"
+msgstr "intro"
+
+#: Game.Levels.Implication.L06intro
+msgid ""
+"## Summary\n"
+"\n"
+"If the goal is `P → Q`, then `intro h` will introduce `h : P` as a hypothesis,\n"
+"and change the goal to `Q`. Mathematically, it says that to prove $P \\implies Q$,\n"
+"we can assume $P$ and then prove $Q$.\n"
+"\n"
+"### Example:\n"
+"\n"
+"If your goal is `x + 1 = y + 1 → x = y` (the way Lean writes $x+1=y+1 \\implies x=y$)\n"
+"then `intro h` will give you a hypothesis $x+1=y+1$ named `h`, and the goal\n"
+"will change to $x=y$."
+msgstr ""
+"## 小结\n"
+"\n"
+"如果目标是 `P → Q`，那么 `intro h` 将引入 `h : P` 作为假设，\n"
+"并将目标更改为 `Q`。从数学上讲，要证明 $P\\implies Q$，\n"
+"我们可以假设 $P$ ，然后证明 $Q$ 。\n"
+"\n"
+"### 例子：\n"
+"\n"
+"如果您的目标是 `x + 1 = y + 1 → x = y` （在Lean中这表示 $x+1=y+1\\implies x=y$ ）\n"
+"那么 `intro h` 会给你一个名为 `h` 的假设 $x+1=y+1$ ，目标\n"
+"也同时更改为 $x=y$。"
+
+#: Game.Levels.Implication.L06intro
+msgid ""
+"We have seen how to `apply` theorems and assumptions\n"
+"of the form `P → Q`. But what if our *goal* is of the form `P → Q`?\n"
+"To prove this goal, we need to know how to say \"let's assume `P` and deduce `Q`\"\n"
+"in Lean. We do this with the `intro` tactic.\n"
+msgstr ""
+"我们已经看到了如何 `apply` 形式为 `P → Q` 的定理和假设。\n"
+"但如果我们的 *目标* 是形式为 `P → Q` 的呢？\n"
+"要证明这个目标，我们需要知道如何在 Lean 中表示 “假设 `P` 并推导出 `Q`”。我们用 `intro` 策略来做这件事。\n"
+
+#: Game.Levels.Implication.L06intro
+msgid "$x=37\\implies x=37$."
+msgstr "$x=37\\implies x=37$ 。"
+
+#: Game.Levels.Implication.L06intro
+msgid "Start with `intro h` to assume the hypothesis and call its proof `h`."
+msgstr "使用 `intro h` 来设假设为 `h`。"
+
+#: Game.Levels.Implication.L06intro
+msgid "Now `exact h` finishes the job."
+msgstr "现在，用 `exact h ` 完成证明。"
+
+#: Game.Levels.Implication.L07intro2
+msgid "intro practice"
+msgstr "入门练习（译注：翻译存疑，没刷过最新版本，有可能是 `intro` 策略的练习。）"
```

Comment:
Can you check where it appears in the UI and remove the translation note?

Reply:
I will do this as soon as the UI is ready.

Reply:
It's the title of [this level](https://adam.math.hhu.de/#/g/leanprover-community/nng4/world/Implication/level/7).